srcpkgs/screen/INSTALL

```diff
@@ -0,0 +1,5 @@
+case "${ACTION}" in
+post)
+	. /etc/runit/core-services/20-screen.sh
```

Comment:
Not sure if this is fine, most `INSTALL` actions would deal with `run/screen` so they affect the target root... Might be better to just copy the `install` command, IMO.

Reply:
(but use `run/` instead of `/run/`

Reply:
done